### DISCUSSION TITLE
Primefaces Mirage Update

### DISCUSSION THREAD

**Discussion by musttafayildirim** [primefaces/community#2047] [PrimeFaces Templates] [answered]
I am using PrimeFaces 8 with the Mirage theme. We have decided to upgrade the PrimeFaces version and are working on it. When we upgraded to PrimeFaces 10, the sorting in DataTables did not work. In higher versions, the sorting in DataTables works, but the icons of the components do not display correctly. I have resolved the issue related to FontAwesome, but the icons in SelectOneMenu appear as squares. Can you share the necessary steps specific to the Mirage theme after updating PrimeFaces?

**Comment by kurtomerfaruk** [comment, marked as answer]
You can follow the changes related to version updates from the link below. As far as I remember, there have been changes in some components. I guess it is not working due to these changes. I hope the migration guide will help you

[https://primefaces.github.io/primefaces/14_0_0/#/../migrationguide/migrationguide](url)
